pyproject.toml

```
@@ -26,7 +26,7 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-  "arches>=7.6.0a1,<8",
+  "arches @ git+https://github.com/archesproject/arches.git@dev/7.6.x",
```

Maybe we should loosen this a little bit.
```suggestion
  "arches>=7.6.2,<8",
```